persistence-api/src/test/java/io/stargate/db/cdc/CDCTestUtil.java

```diff
@@ -0,0 +1,40 @@
+package io.stargate.db.cdc;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.stream.Collectors;
+
+public final class CDCTestUtil {
```

Comment:
Is this class used? I could not find any references to it in this PR :thinking:

Reply:
It's used only by `TableSchemaManagerTest`, it's also a leftover from the port.

I'll move those methods to the test that is actually using it :)

Reply:
I've addressed most of the issues from your review.